src/figuresClasses.ts

```diff
@@ -1,11 +1,72 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const sides = [a, b, c].sort((x, y) => x - y);
+
+    if (sides[2] >= sides[0] + sides[1]) {
+      throw new Error('Sum of smaller sides must be greater then bigger side');
```

Comment:
[CODE STYLE] Good job on throwing an error when the triangle sides don't follow the triangle inequality rule. However, the error message could be more descriptive. For example:

```typescript
throw new Error('In a triangle, the sum of lengths of any two sides must be greater than the length of the third side.')
```
This comment applies to all error messages in your constructors.